controls/imageeditor/toolbar/items.md

```diff
@@ -44,28 +44,28 @@ The table below shows the built-in toolbar items available for desktop - WinUI a
 | `ImageEditorRedoToolbarItem` | Redoes the last executed action on the image in the ImageEditor |
 | `SeparatorToolbarItem` | Represents a separator between the toolbar items |
 
-![ImageEditor ToolbarItems for Desktop](../images/imageeditor-tooolbar-items-desktop.png "ToolbarItems for Mobile")
+![ImageEditor Toolbar Items for Desktop](../images/imageeditor-tooolbar-items-desktop.png "ToolbarItems for Mobile")
 
 ## Toolbar Configuration
 
-`RadImageEditorToolbar` inherits from the `RadToolbar`({%slug toolbar-overview%}). All properties for configuration and styling available for the RadToolbar applies to the RadImageEditorToolbar.
+`RadImageEditorToolbar` inherits from the `RadToolbar`({%slug toolbar-overview%}). All properties for configuration and styling available for the `RadToolbar` applies to the `RadImageEditorToolbar`.
 
 ## Toolbar Item Configuration 
 
-Review all properties availabe for toolbar items configuration.
+Review all properties available for toolbar items configuration.
 
-**Properties for all toolbar items**
+**Properties for All Toolbar Items**
 
-The properties described bellow apply to all tollbar items available in the ImageEditorToolbar:
+The properties described bellow apply to all toolbar items available in the `ImageEditorToolbar`:
 
 * `ControlTemplate`(`Microsoft.Maui.Controls.ControlTemplate` with target type `Telerik.Maui.Controls.ToolbarItem`)&mdash;Specifies the control template of the toolbar item.
 * `IsEnabled`(`bool`)&mdash;Specifies whether the toolbar item is enabled.
 * `IsVisible`(`bool`)&mdash;Specifies whether the toolbar item is visible.
 * `PlacementOptions`(enum of type `Telerik.Maui.Controls.ToolbarItemPlacementOptions`)&mdash;Defines the allowed placement options of the toolbar item in the toolbar. The available options are:
-	* ToolStrip&mdash;The toolbar item appears in the main tool strip area of the toolbar.
-	* DropDown&mdash;The toolbar item appears in the overflow drop-down menu of the toolbar.
+	* `ToolStrip`&mdash;The toolbar item appears in the main tool strip area of the toolbar.
+	* `DropDown`&mdash;The toolbar item appears in the overflow drop-down menu of the toolbar.
 
-**Properties for specific toolbar items**
+**Properties for Specific Toolbar Items**
```

Comment:
Don't use bold. Make this an H3. Apply this change to all occurrences in this file.